src/main/java/uk/gov/companieshouse/ocr/api/image/extracttext/ImageOcrService.java

```diff
@@ -106,11 +114,19 @@ private String extractTextOnCurrentPage(ImageReader reader, TextConversionResult
         var level = TessPageIteratorLevel.RIL_TEXTLINE;
   
         if (ri != null) {
+            var lineNum = 0;
             do {
+                lineNum++;
                 var symbol = api.TessResultIteratorGetUTF8Text(ri, level);
                 var confidence = api.TessResultIteratorConfidence(ri, level);
                 if (symbol != null) {
+                    var lineText = symbol.getString(0);
```

Comment:
Can you put this in a private funtion - I have looked at the com.sun.jna.Pointer class and think we need to be extra safe here since this code will always run.

Do a try catch and if an exception occurs on this then return an empty String